tests/test_manager/test_manager.py

```diff
@@ -357,6 +357,12 @@ def test_start_all(self, *args):
             period=0.5,
         )
 
+        wait_for_condition(
+            lambda: "Start processing messages"
+            in open(self.log_file(self.agent_name), "r").read(),
+            timeout=60,
```

Comment:
Introduce a constant `DEFAULT_TIMEOUT = 60` and use across.

Reply:
done